Misc/NEWS.d/next/Library/2024-12-14-19-51-39.gh-issue-127930.WsGnh9.rst

```diff
@@ -0,0 +1 @@
+Add :attr:`__all__` to :mod:`tkinter.simpledialog`.
```

Comment:
```suggestion
Add ``__all__`` to :mod:`tkinter.simpledialog`.
```

We don't document `__all__` but I think it's worth adding a new glossary term for this one and point it to https://docs.python.org/3/tutorial/modules.html#importing-from-a-package. But for this PR, let's keep it simple.